src/w2grid.js

```diff
@@ -7478,9 +7479,21 @@
                     '></span>';
             }
             // various renderers
+            function getTitle(cellData){
```

Comment:
The only suggestion is to move this function to the end of the parent function. I think it is more readble to have nested functions at the end. Even after `return`. I will do it.